malariagen_data/pf7.py

```diff
@@ -0,0 +1,291 @@
+import json
+import os
+
+import dask.array as da
+import pandas as pd
+import xarray
+import zarr
+
+from malariagen_data.util import (
+    DIM_ALLELE,
+    DIM_PLOIDY,
+    DIM_SAMPLE,
+    DIM_VARIANT,
+    da_from_zarr,
+    init_filesystem,
+    init_zarr_store,
+)
+
+DIM_ALT_ALLELE = "alt_alleles"
+DIM_STATISTICS = "sb_statistics"
+DIM_GENOTYPES = "genotypes"
+
+
+class Pf7:
+    def __init__(
+        self,
+        url,
+        data_config=None,
+        **kwargs,
+    ):
+
+        # setup filesystem
+        self._fs, self._path = init_filesystem(url, **kwargs)
+        self.CONF = self._load_config(data_config)
+
+        # setup caches
+        self._cache_sample_metadata = None
+        self._cache_zarr = None
```

Comment:
Suggest to rename to `_cache_variant_calls_zarr` for clarity.

Reply:
changed this for the cache and also the name of the path in the config